mps_youtube/commands/generate_playlist.py

```diff
@@ -76,3 +85,43 @@ def random_plname():
     n_chars = 6
     return ''.join(choice(string.ascii_lowercase + string.digits)
                    for _ in range(n_chars))
+
+
+def description_generator(text):
+    """ Fetches a videos description and parses it for
+        <artist> - <track> combinations
+    """
+    if not isinstance(g.model, Playlist):
+        g.message = util.F("mkp desc unknown")
+        return
+
+    # Use only the first result, for now
+    num = text.replace("--description", "")
+    num = num.replace("-d", "")
+    num = util.number_string_to_list(num)[0]
+
+    query = {}
+    query['id'] = g.model[num].ytid
+    query['part'] = 'snippet'
+    query['maxResults'] = '1'
+    data = pafy.call_gdata('videos', query)['items'][0]['snippet']
+    title = "mkp %s" % data['title']
+    data = util.fetch_songs(data['description'])
+
+    columns = [
+        {"name": "idx", "size": 3, "heading": "Num"},
+        {"name": "artist", "size": 30, "heading": "Artist"},
+        {"name": "title", "size": "remaining", "heading": "Title"},
+    ]
+
+    def run_m(idx):
+        """ Create playlist based on the
+            results selected
+        """
+        create_playlist(idx, title)
+
+    data = [listview.ListSongtitle(x) for x in data]
+    g.content = listview.ListView(columns, data, run_m)
```

Comment:
Yea, I guess if the software is unable to find any tracks, it would throw exceptions. I'll add a check for that.